torch/csrc/jit/import_source.cpp

```diff
@@ -52,8 +53,12 @@ struct TORCH_API ClassNamespaceValue : public SugaredValue {
       Function& m,
       const std::string& name) override {
     auto fullName = c10::QualifiedName(basename_, name);
-    if (auto classType = cu_.get_class(fullName)) {
-      return std::make_shared<ClassValue>(classType);
+    if (auto serializable_type = cu_.get_serializable_type(fullName)) {
```

Comment:
This is going to conflict with https://github.com/pytorch/pytorch/pull/21853 which also needs to redefine how named types are put in the compilation unit.